examples/store/src/components/product/ProductOptions/ProductOptions.tsx

```diff
@@ -24,7 +24,7 @@ export const ProductOptions: React.FC<ProductOptionsProps> = ({
                     <div role="listbox" className="flex flex-row py-4">
                         {opt.values
                             .filter(
-                                (val, index: number, arr: any[]) =>
+                                (val, index: number, arr: any[]) => // eslint-disable-line
```

Comment:
```suggestion
                                (val, index: number, arr: ProductOptionValue[]) =>
```

We also need to add `ProductOptionValue` import to the `@medusajd/medusa` import line.

Reply:
@aliemir  `ProductOptionValue`  is not found in `@medusajs/medusa`. May it be only `ProductOption`.